web/conf/isil2opac_hbzid.json

```diff
@@ -217,17 +217,20 @@
     "DE-294-140": "https://suchen.ub.rub.de/hbzid/{hbzid}",
     "DE-361": "http://katalogplus.ub.uni-bielefeld.de/cgi-bin/new_search.cgi?kat1=si&var1={hbzid}&kat2=ti&var2=&kat3=aup&var3=&bestand=lok&sprache=GER&art=f&fsubmit=1&vr=1&opt=&pagesize=10",
     "DE-385": "http://primo.kobv.de/primo_library/libweb/action/dlSearch.do?vid=TRI_UB_I&institution=TRI_UB_I&search_scope=localude&bulkSize=10&lang=ger&indx=1&onCampus=false&query=any,contains,{hbzid}",
-	"DE-386": "https://opac.ub.uni-kl.de/F?func=item-cen-2&doc_library=KLU01&doc_001_number={hbzid}",
+	  "DE-386": "https://opac.ub.uni-kl.de/F?func=item-cen-2&doc_library=KLU01&doc_001_number={hbzid}",
     "DE-464": "http://primo.ub.uni-due.de/primo_library/libweb/action/dlSearch.do?vid=UDE&institution=UDE&search_scope=localude&bulkSize=10&lang=ger&indx=1&onCampus=false&query=any,contains,{hbzid}",
     "DE-465": "http://primo.ub.uni-due.de/primo_library/libweb/action/dlSearch.do?vid=UDE&institution=UDE&search_scope=localude&bulkSize=10&lang=ger&indx=1&onCampus=false&query=any,contains,{hbzid}",
+    "DE-386": "https://opac.ub.uni-kl.de/F?func=item-cen-2&doc_library=KLU01&doc_001_number={hbzid}",
+    "DE-464": "http://primo.ub.uni-due.de/primo_library/libweb/action/search.do?fn=search&vl%281UI0%29=exact&vl%28freeText0%29={hbzid}",
+    "DE-465": "http://primo.ub.uni-due.de/primo_library/libweb/action/search.do?fn=search&vl%281UI0%29=exact&vl%28freeText0%29={hbzid}",
```

Comment:
These three added lines have the same ISIL as the lines before. Should they replace them?